docs/how-to-guides/software-management/software-support.md

```diff
@@ -49,4 +67,23 @@ Prioritization meetings are typically a loosely guided discussion in which the t
 1. Ask for any closing remarks
 
 ??? Note "Issue deferral"
-    Deferring an issue generally means "this is a good, important issue, but it requires further discussion." This discussion should not interrupt the flow of prioritization, and discussing deferred issues is usually a good final topic.
\ No newline at end of file
+    Deferring an issue generally means "this is a good, important issue, but it requires further discussion." This discussion should not interrupt the flow of prioritization, and discussing deferred issues is usually a good final topic.
```

Comment:
What does 'a good final topic' mean?

Reply:
I didn't write that, was part of the original doc, I think Im just going to remove that sentence.